page-objects/pages/TASKListPage.js

```diff
@@ -106,12 +106,6 @@ async function clickActionFromMenuOnTask(button) {
 	await elementHelpers.clickAndWait(actionButton);
 }
 
-async function getTaskDescription() {
```

Comment:
Can you please check in your team if they have unit tests to check if the description is shown. If yes, we don't need to check it as well, if not then we should check if it's there.

Reply:
I asked in my team and yes they check it via unit tests.

https://github.com/hpi-schul-cloud/schulcloud-server/blob/55b82f8b553c3962c3733acd498795cc8e1d575a/apps/server/src/shared/domain/entity/task.entity.spec.ts#L577-L591

Reply:
Perfect, thx :)